vipps-psp-api.md

```diff
@@ -101,6 +101,27 @@ through the acquirer and responds to the makePayment-call with the payment
 request status. End user receives confirmation in Vipps app. Vipps redirects
 the end user to the redirectUrl provided during payment initiation.
 
+### Cancelling pending transactions
+
+A user might go back to the PSP's checkout without loggin in to the Vipps App, or aborting it in the Vipps App.
+Vipps's recommendation is that the PSP then cancels the transaction in their backend. And returns error code
```

Comment:
Vipps's recommendation is that the PSP then cancel the transaction in their backend and return error code